pipelines/broad/arrays/single_sample/Arrays.documentation.md

```diff
@@ -1,6 +1,6 @@
 # Arrays Pipeline
 
-This document describes version 2.3.5 of the Arrays Pipeline. The Arrays Pipeline is used internally within the Broad Institute.  It is written in WDL and designed to work on the Cromwell execution engine using Google Cloud Services and Google Cloud Storage as its backend and storage.  
+This document describes version 2.5.2 of the Arrays Pipeline. The Arrays Pipeline is used internally within the Broad Institute.  It is written in WDL and designed to work on the Cromwell execution engine using Google Cloud Services and Google Cloud Storage as its backend and storage.  
```

Comment:
On this branch `Arrays.wdl` says `pipeline_version = "2.5.1"`, should that match the version stated here?

Reply:
Yeah, I will update that.  I had anticipated merging another branch (which had that version), but shouldn't get ahead of myself.